solutions/page_object/pages/base_page.py

```diff
@@ -52,6 +52,11 @@ def should_be_login_link(self):
             BasePageLocators.LOGIN_LINK
         ), "Login link is not presented"
 
+    def should_be_authorized_user(self):
+        assert self.is_element_present(BasePageLocators.USER_ICON), (
+            "User icon is not presented," " probably unauthorised user"
+        )
```

Comment:
А не проще ли так? Или в чём суть двух строк в assertion_message?

```suggestion
        assert self.is_element_present(
            BasePageLocators.USER_ICON
        ), "User icon is not presented, probably unauthorised user"
```